flake8_eradicate.py

```diff
@@ -68,6 +72,31 @@ def add_options(cls, parser: OptionManager) -> None:
             action='store_true',
             parse_from_config=True,
         )
+        parser.add_option(
```

Comment:
These options can be lists, see https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/options/config.py#L235-L248

Reply:
Well I kinda see what you want, but for just 3 options isn't this a bit overkill?
IMHO this would make the code just longer and more complex rather than more readable.

Reply:
I think I was not clear enough. Sorry for that  🙂 

I want these three settings to include `comma_separated_list=True`. This is a standard way to handle things instead of `#` separated list

Reply:
Ah ok. Now I understand.

I made it here just like I made it in eradicate: see https://github.com/myint/eradicate/blob/master/eradicate.py#L268-L271

Because the part from the link needed to be copied to flask-eraddicate (see https://github.com/Cielquan/flake8-eradicate/blob/update-to-new-eradicate-api/flake8_eradicate.py#L45-L52) I can change it to a comma-separated list if you want, but this would change the way how to config the whitelist between basic `eradicate` and `flask-eradicate` which IMO is not that nice/good.

The reason for why it is how it is I wrote above:

> Reason for this is, this way all white-spaces are preserved the user puts in front or after the `#`.

If you still want me to change it I will ;)